spec/index.bs

```diff
@@ -13129,8 +13128,17 @@ be rare and can safely be ignored. Applications should not be written in such a
 timestamps cause an application failure.
 
 <p tracking-vector>
-Timestamp queries can provide high-resolution GPU timing.
-See [[#security-timing]] for security considerations.
+Timestamp queries can provide high-resolution GPU timing. To mitigate privacy concerns, they are
+aligned to a lower precision with the following algorithm:
+
+<div algorithm>
+    To get the <dfn abstract-op>current queue timestamp</dfn>:
+    
+    Return the result of calling [=coarsen time=] with the timestamp value.
```

Comment:
```suggestion
    - Let |fineTimestamp| be the current timestamp value of the current [=queue timeline=],
        in nanoseconds, relative to an implementation-defined point in the past.
    - Return the result of calling [=coarsen time=] on |fineTimestamp|.
```

The "implementation-defined point in the past" thing is intentionally underspecified - it isn't guaranteed to always be the *same* point in the past - filed #4361.